packages/snap-preact-components/src/components/Molecules/FacetHierarchyOptions/FacetHierarchyOptions.stories.tsx

```diff
@@ -93,7 +103,7 @@ const ObservableFacetHierarchyOptions = observer(({ args, controller }: { args:
 		.shift()
 		.values.slice(0, 12);
 
-	return <FacetHierarchyOptions {...args} values={hierarchyValues} />;
+	return <FacetHierarchyOptions {...args} values={hierarchyValues} facet={hierarchyValues} />;
```

Comment:
Should the `facet` prop be here? If so, should it have the same value as the `values` prop?